optuna/study/study.py

```diff
@@ -753,7 +756,10 @@ def objective(trial):
         self._stop_flag = True
 
     def enqueue_trial(
-        self, params: Dict[str, Any], user_attrs: Optional[Dict[str, Any]] = None
+        self,
+        params: Dict[str, Any],
+        user_attrs: Optional[Dict[str, Any]] = None,
+        skip_if_exists: bool = True,
```

Comment:
I prefer that the default value of `skip_if_exists` is `False` because skipping an enqueued trial may not be intuitive for users.

Reply:
a82dbabc7d52b188c9272e8d8e1502f772c2a5da :+1: